.github/workflows/check-pr-title.yaml

```
@@ -0,0 +1,20 @@
+name: "Check PR title for the greater good"
+
+on:
+  pull_request:
```

```suggestion
  pull_request_target:
```

So it will run on external PRs

---

Right, I see, this didn't trigger the event on the PR title change, when I tested it, because the workflow didn't exist in the target.